src/master/about.md

```diff
@@ -31,21 +31,13 @@ immudb can operate both as a key-value or relational (SQL) database. You can add
 
 Depending on your use case, immudb might function as your application's primary or as a secondary database. As a secondary, complimentary database, use immudb to cross-check the integrity of your important data (by verifying checksums or comparing stored data values). A secondary database enables you to quickly use immudb without completely re-engineering your existing application.
 
-### Use cases:
-  - Integration with your DevOps ensures code security throughout the development and deployment process. Embed immudb into your [Azure DevOps](https://codenotary.io/blog/securing-your-azure-devops-ecosystem-jenkins-and-kubernetes-aks/) with Jenkins and Kubernetes. Use just [Jenkins](https://codenotary.io/blog/jenkins-build-deployment-pipeline-a-how-to-for-ensuring-integrity/). Alternatively, integrate with [GitLab](https://codenotary.io/blog/fully-trusted-gitlab-pipeline/) or [GitHub](https://codenotary.io/blog/use-github-actions-for-validated-builds/).
-
-  - Guarantee [File Integrity](https://codenotary.io/blog/file-integrity-monitoring-change-management/) of your critical data. Examples include storing your organization's sensitive financial, credit card transactional, invoices, contracts, educational transcripts, and other important data.
-
-  - Ensure integrity of your legal [Documents and Invoices](https://codenotary.io/blog/immutably-store-or-guarantee-the-immutability-of-documents-and-invoices-for-compliance-reasons/), contracts, forms, and your downloads and emails.
-
-  - Save your Internet of Things (IoT) sensor data as a failsafe plan for loss of data.
-
-  - Keep your investment guidelines or stock market data tamperproof for your investment bank or client financial portfolios.
-
-  - Store important log files to keep them tamperproof to meet regulations like PCI compliance.
-
-  - Protect medical data, test results, or recipes from alteration.
-
-  - Companies use immudb to protect credit card transactions and to secure processes by storing digital certificates and checksums.
+### Use cases
+
+- Integration with your DevOps ensures code security throughout the development and deployment process. Embed immudb into your [Azure DevOps](https://codenotary.io/blog/securing-your-azure-devops-ecosystem-jenkins-and-kubernetes-aks/) with Jenkins and Kubernetes. Use just [Jenkins](https://codenotary.io/blog/jenkins-build-deployment-pipeline-a-how-to-for-ensuring-integrity/). Alternatively, integrate with [GitLab](https://codenotary.io/blog/fully-trusted-gitlab-pipeline/) or [GitHub](https://codenotary.io/blog/use-github-actions-for-validated-builds/).
+- Save your Internet of Things (IoT) sensor data as a failsafe plan for loss of data.
```

Comment:
I would keep file integrity and legal just without links if they are not working anymore.

Reply:
Restored